engine/lib/membrane_rtc_engine/engine.ex

```diff
@@ -593,9 +568,14 @@ defmodule Membrane.RTC.Engine do
 
   @impl true
   def handle_crash_group_down(endpoint_id, ctx, state) do
-    {{:present, endpoint}, actions, state} = handle_remove_endpoint(endpoint_id, ctx, state)
-    dispatch(%Message.EndpointCrashed{endpoint_id: endpoint_id, endpoint_type: endpoint.type})
-    {actions, state}
+    case handle_remove_endpoint(endpoint_id, ctx, state) do
+      {{:present, endpoint}, actions, state} ->
+        dispatch(%Message.EndpointCrashed{endpoint_id: endpoint_id, endpoint_type: endpoint.type})
+        {actions, state}
+
+      {:absent, actions, state} ->
+        {actions, state}
```

Comment:
Probably yes but we have two problems here:
1. We don't have an endpoint, so we don't know its type, so we would have to dispatch the message `EndpointCrashed` with type `:unknown`.
2. If I remember correctly, I added this case because sometimes, after properly removing an endpoint, this was triggered a second time, so firing `EndpointCrashed` here could lead to the backend logic receiving two notifications: first `EndpointRemoved` and then `EndpointCrashed`.

Reply:
Good point, let's leave a log at least. I am not sure whether warning or debug but I think I would opt for a warning?